cloud/setup_env.sh

```diff
@@ -0,0 +1,13 @@
+#/bin/bash
```

Comment:
We use envdir for setting the environment variables to be used by the cloud. It's not well documented, but the script that gets run as part of CloudDeploy pulls a directory of these variables from S3.